### PR TITLE
update markdown tests

### PR DESCRIPTION
**Pull request by TravisEz13**
## PR Summary

update markdown tests
  - explicitly use gulp 4.0 beta because 3.9 depends on a vulnerable package

## PR Checklist

- [x] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - Use the present tense and imperative mood when describing your changes
- [x] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] [Change is not breaking](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)
- [x] [Make sure all `.h`, `.cpp`, `.cs`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
    - If the PR is work in progress, please add the prefix `WIP:` to the beginning of the title and remove the prefix when the PR is ready.
- **User-facing changes**
    - [x] Not Applicable
    - **OR**
    - [ ] User-facing [Documentation needed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
        - [ ] Issue filed - Issue link:
- **Testing - New and feature**
    - [x] Not Applicable or can only be tested interactively
    - **OR**
    - [ ] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)
        - [ ] [Add `[feature]` if the change is significant or affects feature tests](https://github.com/PowerShell/PowerShell/blob/master/docs/testing-guidelines/testing-guidelines.md#requesting-additional-tests-for-a-pr)

### CODE REVIEW

File: test/common/markdown/markdown.tests.ps1 [hunk 1]
```diff
@@ -9,7 +9,7 @@ $repoRootPathFound = $false
 Describe 'Common Tests - Validate Markdown Files' -Tag 'CI' {
     BeforeAll {
         # Skip if not windows, We don't need these tests to run on linux (the tests run fine in travis-ci)
-        $skip = !$IsWindows
+        $skip = $false
```
Comment (rjmholt) [inline test/common/markdown/markdown.tests.ps1]: Is this a temporary change, or should we just take the `if`-statement out?

**Review by rjmholt — APPROVED** [review]
Left a comment, but it's non-blocking.

Otherwise, LGTM

File: test/common/markdown/markdown.tests.ps1 (hunk 1 above)
Comment (TravisEz13) [reply test/common/markdown/markdown.tests.ps1]: removed this change.  It was to allow local testing of the change.

**Review by daxian-dbw — APPROVED** [review]

**Review by anmenaga — APPROVED** [review]

File: test/common/markdown/markdown.tests.ps1 [hunk 2]
```diff
@@ -1,5 +1,7 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License.
+
+Import-Module HelpersCommon
```
Comment (iSazonov) [inline test/common/markdown/markdown.tests.ps1]: Autoloading should works.

**Review by adityapatwardhan — APPROVED** [review]

**Pull request merged**